sanic/response.py

```diff
@@ -73,28 +73,121 @@
 }
 
 
-class HTTPResponse:
+class BaseHTTPResponse:
+    def _encode_body(self, data):
+        try:
+            # Try to encode it regularly
+            return data.encode()
+        except AttributeError:
```

Comment:
Actually looking at it, I don't think there's a situation where the body passed would be in bytes form, since `ujson.dumps` returns a `str`. Bytes should be passed into the response as `body_bytes`, so I think we're okay as it is.